com.microsoft.mrtk.input/Interactors/HandJointInteractor.cs

```diff
@@ -131,15 +131,18 @@ public override void ProcessInteractor(XRInteractionUpdateOrder.UpdatePhase upda
                     interactionPointTracked = TryGetInteractionPoint(out Pose interactionPose);
                     if (interactionPointTracked)
                     {
-                        attachTransform.SetPositionAndRotation(interactionPose.position, interactionPose.rotation);
+                        transform.SetPositionAndRotation(interactionPose.position, interactionPose.rotation);
                     }
                     else
                     {
-                        // If we have no valid tracked interaction point, reset to whatever our parent XRController's pose is.
-                        attachTransform.localPosition = Vector3.zero;
-                        attachTransform.localRotation = Quaternion.identity;
+                        // If we don't have a joint pose, reset to whatever our parent XRController's pose is.
+                        transform.localPosition = Vector3.zero;
+                        transform.localRotation = Quaternion.identity;
                     }
 
+                    // Ensure that the attachTransform tightly follows the interactor's transform
+                    attachTransform.SetPositionAndRotation(transform.position, transform.rotation);
```

Comment:
Could we just set the local position/rotation to zero/identity for this?

Reply:
I reverted that pattern because I think it makes some assumptions about how the attachTransform is placed on the hierarchy relative to the main transform.

Just wanted to be absolutely explicit in case behaviors change or we introduce different behaviors.